docs/src/pages/[platform]/connected-components/liveness/customization/CustomizationCdn.tsx

```diff
@@ -2,7 +2,11 @@ import React from 'react';
 import { FaceLivenessDetector } from '@aws-amplify/ui-react-liveness';
 
 export function CustomizationCdn() {
-  return (
+  const [showLiveness, setShowLiveness] = React.useState(false);
```

Comment:
This component throws an error when trying to render on the server because its using the browser navigator object.  So the `FaceLivenessDetector` is being rendered client side only.